### ISSUE TITLE
Exclude keys with modifiers from specific sites

### ISSUE THREAD

**Issue by wchargin** [philc/vimium#3694]
I want to configure Vimium to pass through `<C-e>` on Google Meet, which
uses that shortcut to toggle video. Specifically, I want to be able to
load a Meet tab and press Ctrl-E to toggle my video without having to
enter insert mode.

I’ve tried setting an exclusion rule for `e`, `E`, `<c-e>`, `<C-e>`, and
`<Ctrl-e>`; none works. For the last three patterns, the top bar
displays text like “Vimium will exclude `-<>Celrt`”, suggesting that
this isn’t being parsed in the way that I want.

I do still want to be able to use `<C-e>` on other sites; I find it very
helpful.

I’ve read related issues like #3168, #2889, and #1906, but these seem to
all be about disabling the `<C-[>` escape sequence specifically.

**Comment by GaetanLepage** [comment]
Hello !
I have the exact same concern.

**Comment by onichandame** [comment]
> Hello ! As I wanted to completely disable it, I simply `unmap`ed it.
> 
> `unmap <c-e>`

false. unmap requires the following word to have only 1 letter e.g. `unmap j`. `unmap <c-e>` is just ignored.

**Comment by gdh1995** [comment]
no, unmap should support composed keys and even key sequences, like:
unmap g0



---Original---
From: ***@***.***&gt;
Date: Wed, Sep 29, 2021 18:02 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [philc/vimium] Exclude keys with modifiers from specific sites (#3694)




  
Hello ! As I wanted to completely disable it, I simply unmaped it.
 
unmap <c-e&gt;
  
false. unmap requires the following word to have only 1 letter e.g. unmap j. unmap <c-e&gt; is just ignored.
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by jerkey** [comment]
I had the same problem with Google Sheets, vimium (in Brave Browser) was eating the Escape key no matter how hard i hammered the key, and it was maddening to not be able to leave editing a cell that I did not mean to edit!

answer: click the vimium icon, type <c-[> (meaning Control-[ aka ESC) and click "add rule" and then "save"

I understand that this may not have been working when this thread was started, but it worked for me today.

**Comment by gdh1995** [comment]
@jerkey The reason your `passKeys` worked is it includes `[`, and I think a single `[` character is enough in you situation.

**Comment by smolin** [comment]
Same here - none of these seem to have any effect in "Keys to exclude": e <c-e> <C-e> <c-E> <C-E> Ctrl-e Ctrl-E

Same for "unmap" followed by those strings.

Please advise. Otherwise, a terrific plugin!
